src/PJ_isea.c

```diff
@@ -139,11 +139,13 @@ struct isea_geo {
     double lon, lat;
 };
 
+#ifdef unused
```

Comment:
Why not remove this entirely? It's still there in the log if needed in the future.

Reply:
OK I'll remove it